frontend/public/style/_forms.scss

```diff
@@ -53,17 +53,18 @@
   }
 }
 
-.co-suggestion-box input:focus,
-.co-text-filter:focus {
-  + .form-control-feedback--keyboard-hint {
-    display: none;
-  }
+.co-text-filter:focus + .form-control-feedback--keyboard-hint {
+  display: none;
 }
 
 .co-text-filter {
   min-width: 270px;
 }
 
+.co-has-feedback--full-width {
```

Comment:
This really should be something like `.co-search__text-filter` and be in `_.search.scss`. It's only used there.